lldb/tools/lldb-dap/Handler/RequestHandler.h

```diff
@@ -356,7 +356,21 @@ class StepInRequestHandler : public RequestHandler<protocol::StepInArguments,
   llvm::Error Run(const protocol::StepInArguments &args) const override;
 };
 
-class StepInTargetsRequestHandler : public LegacyRequestHandler {
+class StepInTargetsRequestHandler
+    : public RequestHandler<
+          protocol::StepInTargetsArguments,
+          llvm::Expected<protocol::StepInTargetsResponseBody>> {
+public:
+  using RequestHandler::RequestHandler;
+  static llvm::StringLiteral GetCommand() { return "stepInTargets"; }
+  FeatureSet GetSupportedFeatures() const override {
+    return {protocol::eAdapterFeatureStepInTargetsRequest};
```

Comment:
I wonder if we should support dynamic features.

A dap server can send the `capabilities` event to update which features are supported.

When we send this event, you only need to tell the client about changed features, not all the possible features.

After we create the target, we could send a `capabilities` event if the target supports this feature. I think this is only supported on x86 platforms at the moment.